Common/CMakeLists.txt

```diff
@@ -15,3 +15,4 @@ add_subdirectory(CCDB)
 add_subdirectory(Tasks)
 add_subdirectory(TableProducer)
 add_subdirectory(Tools)
+add_subdirectory(Tools/Multiplicity)
```

Comment:
Please add this to the CMakeLists.txt in Tools instead

Reply:
Thanks, @jgrosseo, done! (was busy preparing LHCP talk)